scripts/publish.sh

```diff
@@ -47,9 +47,10 @@ for i in ${EXTENSIONS[@]}; do
 
   cd "$REPO_ROOT/$i"
   set +e
-  firebase ext:dev:publish $PUBLISHER_ID/$i --non-interactive --force
-  set -e
+  firebase ext:dev:publish $PUBLISHER_ID/$i $FIREX_PUBLISH_EXTRA_FLAGS --non-interactive --force
```

Comment:
how is FIREX_PUBLISH_EXTRA_FLAGS set exactly?

Reply:
the caller of this script can optionally define this env var and if defined it will be passed along.